app/src/main/java/com/cp/tasky/ui/sharecomponent/Next6DaysSelector.kt

```diff
@@ -0,0 +1,92 @@
+package com.cp.tasky.ui.sharecomponent
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import com.cp.tasky.ui.theme.LARGE_16_DP
+import com.cp.tasky.ui.theme.MEDIUM_8_DP
+import com.cp.tasky.ui.theme.MEDIUM_LARGE_12_DP
+import com.cp.tasky.ui.theme.SMALL_4_DP
+
+typealias dayOfWeek = String
+typealias dayOfMonth = String
```

Comment:
Since a typealias represents a specific class, it should also follow the class naming conventions and start with a capital letter: `DayOfWeek`

However, even with that naming, it's not really intuitive to me that this represents a string. When I read day of week, I think of a number